### DISCUSSION TITLE
英文识别模型推理精度异常

### DISCUSSION THREAD

**Discussion by zengzixuan2024** [PaddlePaddle/PaddleOCR#13453] [Q&A]
使用ocr_v4训练英文识别模型，训练完模型后使用eval.py验证模型，模型在验证集的精度为43%，而使用infer_rec.py进行预测，模型输出的结果完全不正确。使用export.py将模型导出为推理模型后，使用predict_rec.py进行预测，模型输出的结果仍然不正确。

**Comment by UserWangZz** [comment]
看一看是不是字典使用错了

**Reply by zengzixuan2024** [reply]
字典是正确的，以下是模型识别的一个例子。
![屏幕截图 2024-07-23 130619](https://github.com/user-attachments/assets/a287bd18-e052-4d1e-a11f-37050205c095)

**Reply by UserWangZz** [reply]
能具体说一下问题嘛？我没有理解你的问题是什么

**Reply by zengzixuan2024** [reply]
你好，我的问题是用ppocrv4训练TAL数据集，精度只有8%，请问有什么方法可以改进识别精度吗

**Reply by UserWangZz** [reply]
你的数据集数据量是多少呢？

**Reply by zengzixuan2024** [reply]
TAL数据集是一个手写英语句子的数据集，数据量为10000张。
数据集图片如下所示：
![1016_1_5](https://github.com/user-attachments/assets/ed63608a-90f9-416c-a352-2c0ac5dd240f)

**Comment by UserWangZz** [comment]
一个可能的原因，可能是图像长宽比太大了，导致resize后在宽度方向损失过于明显。

**Reply by zengzixuan2024** [reply]
那应该怎么做才能解决这个问题

**Reply by UserWangZz** [reply]
简单做个实验尝试从中间切分图片看看，模型识别能力是否work

**Reply by zengzixuan2024** [reply]
好的。请问可以不对图片进行resize吗

**Reply by UserWangZz** [reply]
识别网络输入的大小是固定的，因此会在预处理阶段对图像进行resize
